src/lib/openapi/spec/environment-project-schema.ts

```diff
@@ -63,11 +65,21 @@ export const environmentProjectSchema = {
             description:
                 'Indicates whether the environment can be enabled for feature flags in the project',
         },
+        requiredApprovals: {
```

Comment:
since all the code for envs in global scope and in project scope share all the code it's easier to add it here than to have special case omit for project envs